scripts/upgrade

```diff
@@ -86,17 +90,29 @@ then
   source ./bundle_app
 fi
 
+#=================================================
+# ADVERTISE SERVICE IN ADMIN PANEL
+#=================================================
+yunohost service add $app.target \
+  --log $final_path/diaspora/log/production.log \
+        $final_path/diaspora/log/unicorn-stderr.log\
+        $final_path/diaspora/log/unicorn-stdout.log\
+        $final_path/diaspora/log/sidekiq.log\
+  --description "Diaspora service (unicorn web and sidekiq)"
```

Comment:
are you sure this is necessary? It is already done at install time.

Reply:
Cf linter warning: https://ci-apps.yunohost.org/ci/job/525

Reply:
I guess this should be in ./create_services

Reply:
@ericgaspar ping for the above comment ^ 😄 What do you think?

Reply:
From what I understand the CI expects to see this in the *install*, *upgrade* and *restore* script. I'm not sure if it's a good idea to put it in the `creat_services` file ... but it's up to you

Reply:
Answering because i'm responsible for designing the warning eric mentions :sweat_smile:  : 

Eeeer yeah it could, though package_linter will like it more if you keep this instruction in the install/upgrade/restore script directly because it just does a "dirty" parsing/greping of stuff ... Though that brings again the legitimate question of "but duplicating code is blasphemy!" and indeed it is, but that's how we roll for now and hopefully someday™ in 2021 we'll have a new, better packaging format to reduce the madness involved in packaging apps and duplicating stuff all over the place ...

(Slightly related, I see in the `create_services` file the manipulation using `install` and `ynh_replace_string` - hopefully the new helper `ynh_add_config` from 4.1, which we hope to generalize a bit more in 4.2, should be helpful in reducing the complexity - though not sure why `ynh_add_systemd_config` wasn't used yet ?)